api/src/main/java/org/openmrs/module/eptsreports/reporting/utils/EptsReportUtils.java

```diff
@@ -72,4 +80,40 @@ public static void setupReportDefinition(ReportManager reportManager) {
 		ReportManagerUtil.setupReport(reportManager);
 	}
 	
+	/**
+	 * @param parameterizable
+	 * @param mappings
+	 * @param <T>
+	 * @return
+	 */
+	public static <T extends Parameterizable> Mapped<T> map(T parameterizable, String mappings) {
+		if (parameterizable == null) {
+			throw new NullPointerException("Programming error: missing parameterizable");
```

Comment:
Would be better to use an [IllegalArgumentException](https://docs.oracle.com/javase/8/docs/api/java/lang/IllegalArgumentException.html) here and add message like "parameterizable cannot be null". A NPE in this case is misleading as it'll make you think the caller "dereferenced" a null if you see it in a stack trace.